src/Application/Economy/Commands/BalanceTransactionCommand.cs

```diff
@@ -0,0 +1,31 @@
+using Application.Services;
+using Application.Users.Commands;
+using Domain.Aggregates;
+using Domain.Abstractions;
+using Microsoft.EntityFrameworkCore;
+using FluentValidation;
+using MediatR;
+namespace Application.Economy.Commands;
+
+
+public sealed record BalanceTransactionCommand(User sender, User reciver, decimal amount) : IRequest<bool>;
+
+public sealed class TransactionValidator : AbstractValidator<BalanceTransactionCommand>
+{
+    public TransactionValidator()
+    {
+        RuleFor(x => x.amount).LessThanOrEqualTo(0).WithMessage("amount can not be 0 or less");
```

Comment:
you could write fluent validation rules like linq queries - on multiple lines, its easier to read.

also: in most cases, you dont need custom messages, because fluent validation does this for you.

also: you should have MoreThan 0, the amount should be non negative positive decimal.